distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -564,6 +568,32 @@ protected Option installStartupFile(InputStream resourceInputStream, String dest
         return replaceConfigurationFile(destination, tempFile);
     }
 
+
+    /**
+     * Turns off all logging.
+     *
+     * @throws IOException
+     */
+    public static void turnOffLogging(ConfigurationAdmin configAdmin) throws IOException {
```

Comment:
configAdmin is already available within AbstractIntegrationTest so I don't think this argument is not needed